exporter/awskinesisexporter/exporter.go

```diff
@@ -55,24 +56,9 @@ func (e Exporter) Shutdown(context.Context) error {
 
 // ConsumeTraces receives a span batch and exports it to AWS Kinesis
 func (e Exporter) ConsumeTraces(_ context.Context, td pdata.Traces) error {
-	pBatches, err := jaegertranslator.InternalTracesToJaegerProto(td)
+	err := e.ew.WriteTraces(td)
 	if err != nil {
-		e.logger.Error("error translating span batch", zap.Error(err))
-		return consumererror.Permanent(err)
-	}
-	// TODO: Use a multi error type
-	var exportErr error
-	for _, pBatch := range pBatches {
-		for _, span := range pBatch.GetSpans() {
-			if span.Process == nil {
-				span.Process = pBatch.Process
-			}
-			err := e.awskinesis.ExportSpan(span)
-			if err != nil {
-				e.logger.Error("error exporting span to awskinesis", zap.Error(err))
-				exportErr = err
-			}
-		}
+		e.logger.Error("Unable to write traces to kinesis", zap.Error(err))
```

Comment:
I think the pipeline should report errors back to the user so instead of logging, you might want to wrap the error with a helpful message and return it.

Reply:
Is it acceptable to just not log at all? 

My intentions where to keep the original flow from the function and that was to log when there is an error and return the error.

Reply:
Sorry, I reread what had written, @owais.

I had gone removed the logging and leaving it for the pipeline while implementing the todo that was original there.